spring-cloud-aws-sqs/src/test/java/io/awspring/cloud/sqs/integration/BaseSqsIntegrationTest.java

```diff
@@ -93,7 +95,14 @@ protected static CompletableFuture<?> createQueue(SqsAsyncClient client, String
 			if (purgeQueues) {
 				String queueUrl = v.queueUrl();
 				logger.debug("Purging queue {}", queueName);
-				return client.purgeQueue(req -> req.queueUrl(queueUrl).build());
+				return client.purgeQueue(req -> req.queueUrl(queueUrl).build())
+					.thenRun(() -> {
+						if (waitForPurge) {
+							logger.info("Waiting 30000 seconds to start sending.");
```

Comment:
30000 seconds seems like quite long ;)

Reply:
Whoops... I do wish I made this mistake in the logs less often, but it just comes naturally apparently 🙂 

I've fixed this and changed a couple default test settings.

Anything else you see?

Thanks.

Reply:
it looks good to me now 🤌